src/state_point.cpp

```diff
@@ -818,6 +818,13 @@ void write_unstructured_mesh_results()
       if (!umesh->output_)
         continue;
 
+      if (umesh->library() == "moab") {
+        if (mpi::master)
+        warning(fmt::format("Output for a MOAB mesh (mesh {}) was 
+        requested but will not be written. Please use the Python API to generated the desired VTK tetrahedral mesh.", umesh->id_));
+        continue;
```

Comment:
Formatting still looks messed up to me. Do you have a [post-commit hook](https://docs.openmc.org/en/latest/devguide/styleguide.html#automatic-formatting) for clang-format setup?

Reply:
I think I've addressed this. Let me know if it still looks messed up! I do have the post-commit hook installed. Not sure why it didn't apply here...

Reply:
I notice there is a pre-commit option for clang if anyone is interested https://github.com/pre-commit/mirrors-clang-format